pkg/eyecandy/emoji.go

```diff
@@ -0,0 +1,26 @@
+package eyecandy
+
+import (
+	"fmt"
+	"github.com/kyokomi/emoji/v2"
+	"regexp"
+)
+
+func ESPrintf(emojisDisabled bool, format string, v ...interface{}) string {
+	if emojisDisabled {
+		return fmt.Sprintf(removeEmojiFromString(format), v...)
+	}
+	return emoji.Sprintf(format, v...)
+}
+
+func ESPrint(emojisDisabled bool, s string) string {
+	if emojisDisabled {
+		return fmt.Sprint(removeEmojiFromString(s))
+	}
+	return emoji.Sprint(s)
+}
+
+func removeEmojiFromString(s string) string {
+	re := regexp.MustCompile(`:[a-zA-Z0-9-_+]+?:`)
```

Comment:
Not a showstopper for now (and I'm half guilty of this anyways :sweat_smile:), but I wonder of the speed cost of the lookahead on the regexp, every time we call it on logs.